tortoise/contrib/test/__init__.py

```diff
@@ -92,6 +107,21 @@ class SimpleTestCase(_TestCase):
 
     Based on `asynctest <http://asynctest.readthedocs.io/>`_
     """
+    use_default_loop = True
+
+    def __init_loop(self):
+        if self.use_default_loop:
+            self.loop = _LOOP
+            loop = None
+        else:
+            loop = self.loop = asyncio.new_event_loop()
+
+        policy = _Policy(asyncio.get_event_loop_policy(),
+                         loop, self.forbid_get_event_loop)
+
+        asyncio.set_event_loop_policy(policy)
+
+        self.loop = self._patch_loop(self.loop)
```

Comment:
I can't really get a hold of this function. Where does it get called? And why is has to be with with double underscore ?

Reply:
I'm overriding the function in `asynctest/case.py` (that is where the name comes from).
The only change is to prefer the `_LOOP` that I saved in initalizer(), so that if a custom loop was passed in, it would be used instead of `asyncio.get_event_loop()`.

Reply:
https://github.com/Martiusweb/asynctest/blob/master/asynctest/case.py Are talking about this file?
Shouldn't it then be just one underscore (like `_init_loop`, not `__init_loop`)?

Reply:
:man_facepalming:
You are right.
How did that happen...